OpenChange/MAPIStoreMailMessage.m

```diff
@@ -572,28 +572,21 @@ - (int) getPidTagMessageClass: (void **) data
   return MAPISTORE_SUCCESS;
 }
 
-/* Note: this applies to regular mails... */
-// - (int) getPidTagReplyRequested: (void **) data // TODO
-//                    inMemCtx: (TALLOC_CTX *) memCtx
-// {
-//   if (!headerSetup)
-//     [self _fetchHeaderData];
-
-//   return (mailIsEvent
-//           ? [self getYes: data inMemCtx: memCtx]
-//           : [self getNo: data inMemCtx: memCtx]);
-// }
-
-/* ... while this applies to invitations. */
-- (int) getPidTagResponseRequested: (void **) data // TODO
-                          inMemCtx: (TALLOC_CTX *) memCtx
+- (int) getPidTagReplyRequested: (void **) data
+		       inMemCtx: (TALLOC_CTX *) memCtx
 {
   if (!headerSetup)
     [self _fetchHeaderData];
 
   return (mailIsEvent
```

Comment:
this boolean includes the embedded messages, so we have to add a new property to know if the mail is a meeting request...